src/Eppie.App/Eppie.App.Shared/App.xaml.cs

```diff
@@ -88,7 +88,7 @@ private async void CreateAIService()
         {
             try
             {
-                AIService = new AIService();
+                AIService = new AIService(Core);
```

Comment:
Core  can be recreated at method `OnWipeAllDataNeeded`

Reply:
fixed